packages/sparkling-controls/src/ScrollView.svelte

```diff
@@ -11,12 +11,28 @@
 	//
 	// See the License for the specific language governing permissions and
 	// limitations under the License.
+
+	import { afterUpdate } from 'svelte'
+
 	export let stylez: string = ''
 	export let scrollPosition: number = 0
+	export let autoscroll: boolean = false
+
+	let container: HTMLElement
+
+	afterUpdate(() => {
```

Comment:
Why not use $: { ... }

Reply:
Approach with afterUpdate() was in Svelte tutorial, I'll try with $: (seems to be more correct)

Reply:
Changed to `$:` to correctly update by 'scrollPosition' in CompletionPopup. Autoscrolling still needs to be run in afterUpdate().